test/services/database-analyzer/database-analyzer-sequelize.js

```diff
@@ -35,6 +36,31 @@ describe('Database analyser > Sequelize', () => {
         await sequelizeHelper.close();
       });
 
+      it('should give us constraint_name, table_name_ column_type, column_name, foreign_table_name, foreign_column_name, unique_indexes', async () => {
+        const tableForeignKeysAnalyzer = new TableForeignKeysAnalyzer(databaseConnection, 'public');
+        const constraints = await tableForeignKeysAnalyzer.perform('addresses');
+
+        if (dialect === 'postgres') {
```

Comment:
We are in a loop on ['postgres', 'MySQL'] here. It's sad to have if..else.. on it.
This test is not a Sequelize test (Because it's specific on each DB)
It's better to have a new `describe('Database analyser > Postgres specific')` beside Sequelize tests.

Reply:
In the following PRs it will be used on each db, it is a temporary if, that's why I put it like this.

Reply:
I'm gonna let the if, but I will make a category for query tests